hibernate-jpa/build.gradle

```diff
@@ -7,7 +7,7 @@ dependencies {
 
     api(libs.managed.hibernate.core)
     api(libs.managed.jakarta.transaction.api)
-    api(mnValidation.micronaut.validation)
+    implementation(mnValidation.validation) // jakarta.validation:jakarta.validation-api
```

Comment:
Why do we need to include the validation anyway?

Reply:
I don't think this will fix the issue

Reply:
@dstepanov We need the dependency to `jakarta.validation:jakarta.validation-api` because of [ValidatorFactorySettingSupplier](https://github.com/micronaut-projects/micronaut-sql/blob/master/hibernate-jpa/src/main/java/io/micronaut/configuration/hibernate/jpa/conf/settings/internal/ValidatorFactorySettingSupplier.java). 

@graemerocher 
> I don't think this will fix the issue

In this PR for `hibernate-jpa`build I included: 

`testImplementation(mnHibernateValidator.micronaut.hibernate.validator)`

It seems to get rid of the `Multiple possible bean candidates found: [DefaultInternalConstraintValidatorFactory, DefaultConstraintValidatorFactory]` error.

Reply:
Put it compile only

Reply:
done https://github.com/micronaut-projects/micronaut-sql/pull/1267/commits/09609a2793f82bf6ea335a4d4e29ab72765c7df9